draft-ietf-rats-architecture.md

```diff
@@ -412,8 +420,119 @@ implicitly trusted is often referred to as a Root of Trust.
+policy might be to simply authorize any party supplying a compliant Attestation Result
+signed by a trusted Verifier.  A more complex policy might also entail comparing information
+provided in the result against known-good reference values, or applying more complex logic
+such information. 
+
+Thus, Attestation Results often need to include detailed information about the Attester,
+for use by Relying Parties, much like physical passports and drivers licenses include
+personal information such as name and date of birth.  Unlike Evidence, which is often
+very device- and vendor-specific, Attestation Results can be vendor-neutral if the Verifier
+has a way to generate vendor-agnostic information based on evaluating vendor-specific
+information in Evidence.  This allows a Relying Party's Appraisal Policy to be simpler,
+potentially based on standard ways of expressing the information, while still allowing
+interoperability with heterogeneous devices.
+
+Finally, whereas Evidence is signed by the device (or indirectly by a manufacturer, if
+Endorsements are used), Attestation Results are signed by a Verifier, allowing a Relying
+Party to only need a trust relationship with one entity, rather than a larger set of
+entities, for purposes of its Appraisal Policy.
+
+# Claims Encoding Formats
+
+The following diagram illustrates a relationship to which attestation is desired to be added:
+
+~~~~
+   +-------------+               +-------------+
+   |             |-------------->|             |
+   |  Attester   |  Access some  |   Relying   | Evaluate request
+   |             |    resource   |    Party    | against security policy
+   +-------------+               +-------------+
+~~~~
+{: #clientserver title="Typical Resource Access"}
+
+In this diagram, the protocol between Attester and a Relying Party
+can be any new or existing protocol (e.g., HTTP(S), COAP(S),
+802.1x, OPC UA, etc.), depending on the use case.  Such
+protocols typically already have mechanisms for passing security
+information for purposes of authentication and authorization.  Common
+formats include JWTs {{?RFC7519}}, CWTs {{?RFC8392}}, and X.509 certificates.
+
+To enable attestation to be added to existing protocols, enabling a higher
+level of assurance against malware for example, it is important that
+information needed for evaluating the Attester be usable with existing
+protocols that have constraints around what formats they can transport.
+For example, OPC UA {{OPCUA}} (probably the most common protocol in
+industrial IoT environments) is defined to carry X.509 certificates and so
+security information must be embedded into an X.509 certificate to be passed
+in the protocol.  Thus, attestation-related information could be natively
+encoded in X.509 certificate extensions, or could be natively encoded in
+some other format (e.g., a CWT) which in turn is then encoded in an X.509
+certificate extension. 
```

Comment:
Who/where are we going to define these extensions?  Is it the job of RATS?   Does EAT do the work?  Or OPCUA?  Or?